packages/build-tools/src/index.js

```diff
@@ -1 +1 @@
-export { writeDepVersionFile } from './formVersionWriter';
\ No newline at end of file
+export { writeDepVersionFile } from './formVersionWriter.js';
```

Comment:
is the `.js` necessary?

Reply:
Potentially not, but in the past we have needed to add the `.js` at the end of imports/exports to play nicely with SSR support. When people using SSR environments have consumed our components in the past, they have encountered errors because the file extension was not present.